### PR TITLE
5.0p4 release notes

### PR DESCRIPTION
**Pull request by rrelyea**
@karann-msft @anangaur 
/cc @nkolev92 @zivkan @heng-liu @dominoFire

### CODE REVIEW

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[ace60c6](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1331/commits/ace60c6906e74e8d6180f09205cad3064e37180c)_: 

:clock10: **Preparing**: average preparing time is _41 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[ace60c6](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1331/commits/ace60c6906e74e8d6180f09205cad3064e37180c)_: 

:clock10: **Incremental building**: average incremental building time is _21 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[ace60c6](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1331/commits/ace60c6906e74e8d6180f09205cad3064e37180c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-5.0-preview.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-50p4ReleaseNotes/docs/release-notes/NuGet-5.0-preview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.0-preview?branch=pr-en-us-1331) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C1%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201903010114084613-1331%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2FYPsbuiDUTu8wlM5EJIlM9sNiP20U0qZ8%2BnbQ9pnrW4%3D&st=2019-03-01T01%3A10%3A09Z&se=2019-04-01T01%3A15%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[eeb74fc](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1331/commits/eeb74fc1bb9e8ca7c51d2d0c282cbc09317e4920)_: 

:clock10: **Preparing**: average preparing time is _41 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[eeb74fc](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1331/commits/eeb74fc1bb9e8ca7c51d2d0c282cbc09317e4920)_: 

:clock10: **Incremental building**: average incremental building time is _21 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[eeb74fc](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1331/commits/eeb74fc1bb9e8ca7c51d2d0c282cbc09317e4920)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-5.0-preview.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-50p4ReleaseNotes/docs/release-notes/NuGet-5.0-preview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.0-preview?branch=pr-en-us-1331) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C1%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201903010138517120-1331%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=0sqNEqrxeGcjeinUNe0kHAw7d3BJbRSOgrc166GJYbg%3D&st=2019-03-01T01%3A34%3A51Z&se=2019-04-01T01%3A39%3A51Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/release-notes/NuGet-5.0-preview.md [hunk 1]
```diff
@@ -9,11 +9,57 @@ ms.topic: conceptual
 
 # NuGet 5.0 Preview Release Notes
 
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version| Available in .NET SDK(s)|
+|:---|:---|:---|
+| [**5.0.0-p4**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.0.0-preview4/rc](https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes-preview) | [2.1.600, 2.2.200](https://www.microsoft.com/net/download/visual-studio-sdks) |
+
```
Comment (karann-msft) [inline docs/release-notes/NuGet-5.0-preview.md]: For the .NET SDK, those versions are not available on the page you point to.  I updated the VS link to point to the preview release notes that also has the download link for VS preview.

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[fcb4ac7](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1331/commits/fcb4ac7b5e9a76f52304bbe425758c32cf12052e)_: 

:clock10: **Preparing**: average preparing time is _41 sec(s)_

File: docs/release-notes/NuGet-5.0-preview.md [hunk 2]
```diff
@@ -9,11 +9,59 @@ ms.topic: conceptual
...
+* limit http request number per source through NuGet.Config - [#4538](https://github.com/NuGet/Home/issues/4538)
+
+* NuGet should target Net472 (to help Cleanup the 16.0 build of the VSIX) - [#7143](https://github.com/NuGet/Home/issues/7143)
+
+* PMC: Remove OpenPackagePage command - [#7384](https://github.com/NuGet/Home/issues/7384)
+
+* Make NetCoreApp 3.0 map to NetStandard 2.1 - [#7762](https://github.com/NuGet/Home/issues/7762)
+
+* Add netstandard2.0 support to NuGet.* packages - [#6516](https://github.com/NuGet/Home/issues/6516)
```
Comment (nkolev92) [inline docs/release-notes/NuGet-5.0-preview.md]: I added the .NET Standard 2.0 move in DCRs.

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[fcb4ac7](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1331/commits/fcb4ac7b5e9a76f52304bbe425758c32cf12052e)_: 

:clock10: **Incremental building**: average incremental building time is _21 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[fcb4ac7](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1331/commits/fcb4ac7b5e9a76f52304bbe425758c32cf12052e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-5.0-preview.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-50p4ReleaseNotes/docs/release-notes/NuGet-5.0-preview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.0-preview?branch=pr-en-us-1331) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C1%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201903010144095282-1331%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=GLoseXY%2Bmlp3%2BuYFDfU5N35mV8grUhxr8aopzGKw%2BMM%3D&st=2019-03-01T01%3A40%3A12Z&se=2019-04-01T01%3A45%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[742b78f](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1331/commits/742b78f02e64622f83bad31fd44a7e374876e20f)_: 

:clock10: **Preparing**: average preparing time is _41 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[742b78f](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1331/commits/742b78f02e64622f83bad31fd44a7e374876e20f)_: 

:clock10: **Incremental building**: average incremental building time is _21 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[742b78f](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1331/commits/742b78f02e64622f83bad31fd44a7e374876e20f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-5.0-preview.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-50p4ReleaseNotes/docs/release-notes/NuGet-5.0-preview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.0-preview?branch=pr-en-us-1331) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C1%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201903010201422025-1331%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=UbH3c5jyl0eNW1%2B1DPuqzCQao%2Ffze9ZbSyZfL9Ih3gc%3D&st=2019-03-01T01%3A57%3A45Z&se=2019-04-01T02%3A02%3A45Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**